concrete/src/Config/FileLoader.php

```diff
@@ -132,13 +132,15 @@ public function load($environment, $group, $namespace = null)
 
             $paths = array(
                 "{$path}/generated_overrides/{$group}.php",
+                "{$path}/generated_overrides/{$environment}.{$group}.php",
```

Comment:
Can you describe a situation where you'd end up with an environment namespaced generated_override file?